### PR TITLE
Update settings.json

### PR DESCRIPTION
**Pull request by userdocs**
We can define some settings if using https://marketplace.visualstudio.com/items?itemName=foxundermoon.shell-format

Tabs at 4 spaces.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -6,4 +6,6 @@
         "swizzin",
         "xenial"
     ]
-}
\ No newline at end of file
+    "shellformat.flag": "-i 4",
+    "shellformat.path": ""
```
Comment (flying-sausages) [inline .vscode/settings.json:10]: This should be left out as that's relative for everyone. Leaving this unset will use plugin default, which either downloads the binary or uses it if it is in the path

Comment (userdocs) [reply .vscode/settings.json:10]: we are just talking about line 10? ` "shellformat.path": ""`

**Comment by liaralabs** [comment]
> Tabs at 4 spaces.

Shots fired.

**Review by flying-sausages — APPROVED** [review]
I have nothing else to add to this

**Pull request merged**